tox.ini

```diff
@@ -1,21 +1,22 @@
 [tox]
 envlist =
   docs
-  py27-sphinx{179,185}
-  py{36,37}-sphinx{179,185,201,210}
+  py27-sphinx{17,18}
+  py{36,37}-sphinx{17,18,20,21,22}
 
 [testenv]
 deps =
   pytest
   pdbpp
   .
-  sphinx179: sphinx==1.7.9
-  sphinx185: sphinx==1.8.5
-  sphinx201: sphinx==2.0.1
-  sphinx210: sphinx==2.1.0
+  sphinx17: sphinx==1.7.9
+  sphinx18: sphinx==1.8.5
+  sphinx20: sphinx==2.0.1
+  sphinx21: sphinx==2.1.2
+  sphinx22: sphinx==2.2.0
```

Comment:
A little easier and not a problem with this PR, but these specifiers could be more general:

```
sphinx17: sphinx<1.8
sphinx18: sphinx<1.9
```

etc. This removes the need to update these versions manually and catches backport releases and new releases if we're falling behind and not updating.